### ISSUE TITLE
Request: Fetching resolvers over encrypted query...

### ISSUE THREAD

**Issue by Horizonbli** [DNSCrypt/dnscrypt-proxy#2362]
I couldn't find anything similar.

That said, as it is now, dnscrypt-proxy only fetches resolvers and relays lists over 53.

Isn't it possible to have dnscrypt-proxy fetch those lists, only after establishing the connection to the dnscrypt/doh/quic DNS servers, and not before?

Thank you

**Comment by Horizonbli** [comment]
Not my fault there is no template. No template appeared when I pressed "New issue".

Seriously? A bot, not so clever, running the shots around?

**Comment by Horizonbli** [comment]
> Not my fault there is no template. No template appeared when I pressed "New issue".
> 
> Seriously? A bot, not so clever, running the shots around?

" Open blank issue" has no template provided. That's what I used.

Why is it there, at all? Misleading...
